assignments/this.js

```diff
@@ -1,26 +1,58 @@
 /* The for principles of "this";
-* in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
-* write out a code example of each explanation above
-*/
+ * in your own words. explain the four principle for the "this" keyword below.
+ *
+ * 1. Window Binding is also known is global binding. 'this' refers to the global (aka window) object
+ * 2. Implicit Binding points to the object on which the function is called. In other words, it points to what's to the left of the period when the function is called
+ * 3. New Binding uses the 'new' keyword and constructs a new object. 'this' points to what is being constructed
+ * 4. With Explicit Binding, we can tell the Javascript engine to make 'this' point to a certain value using call(), apply(), or bind()
+ *
+ * write out a code example of each explanation above
+ */
 
 // Principle 1
-
 // code example for Window Binding
+function globalBinding() {
+    console.log(this); //points to the window object
+}
+// globalBinding();
 
 // Principle 2
-
 // code example for Implicit Binding
+function logDrake() {
```

Comment:
i noticed you didn't use the "this" keyword with your Implicit Binding.. Basically with the example provided below, "this" when bound Implicitly, would point to a key within that scope block. So basically. the ${this.Introduction}, points to the introduction key within the myExample object scope.  The value of that interpolation would be : 'Hey man! :)'

// Principle 2

// code example for Implicit Binding
const myExample = {
    Introduction: 'Hey man! :)',
    Introduce: function (name) {
        console.log(`${this.Introduction} I am ${name}`);
    }
};
myExample.Introduce('Josh') // 'Hey man! :) I am Josh'

Reply:
it points outside the Introduce function block, but WITHIN the myExample object :)